server/service/src/sync/central_data_synchroniser.rs

```diff
@@ -1,45 +1,26 @@
-use crate::sync::{
-    sync_api_v5::CentralSyncBatchV5,
-    translation_central::{import_sync_records, TRANSLATION_RECORDS},
-    SyncApiV5, SyncConnectionError,
-};
+use crate::sync::{sync_api_v5::CentralSyncBatchV5, SyncApiV5, SyncConnectionError};
 use chrono::Utc;
 use log::info;
 use repository::{
-    EqualFilter, KeyValueStoreRepository, KeyValueType, RepositoryError, StorageConnection,
-    SyncBufferAction, SyncBufferFilter, SyncBufferRepository, SyncBufferRow,
-    SyncBufferRowRepository,
+    KeyValueStoreRepository, KeyValueType, RepositoryError, StorageConnection, SyncBufferAction,
+    SyncBufferRow, SyncBufferRowRepository,
 };
 use thiserror::Error;
 
-use super::{sync_api_v5::CentralSyncRecordV5, SyncImportError};
+use super::sync_api_v5::CentralSyncRecordV5;
 
 #[derive(Error, Debug)]
-pub enum CentralSyncError {
-    #[error("Failed to pull central sync records - {source:?}")]
-    PullCentralSyncRecordsError { source: SyncConnectionError },
-    #[error("Failed to update central sync buffer records - {source:?}")]
-    UpdateCentralSyncBufferRecordsError { source: RepositoryError },
-    #[error("Failed to get central sync cursor record - {source:?}")]
-    GetCentralSyncCursorRecordError { source: RepositoryError },
-    #[error("Failed to get central sync buffer records - {source:?}")]
-    GetCentralSyncBufferRecordsError { source: RepositoryError },
-    #[error("Failed to import central sync buffer records - {source:?}")]
-    ImportCentralSyncRecordsError { source: SyncImportError },
-    #[error("Failed to remove central sync buffer records - {source:?}")]
-    RemoveCentralSyncBufferRecordsError { source: RepositoryError },
-    #[error("Failed to connect to DB - {source:?}")]
-    DBConnectionError { source: RepositoryError },
-}
-
-impl CentralSyncError {
-    pub fn from_database_error(e: RepositoryError) -> Self {
-        CentralSyncError::DBConnectionError { source: e }
-    }
+pub(crate) enum CentralSyncError {
```

Comment:
one reason I want us to move to pub(crate), is seeing `dead code` (i think i cause the dead code here, but compiler helped when vis is crate wide)

Reply:
I was hoping it would, i made this script that goes through all of the files and replaces visibility one line at a time (doing `cargo check` check in between), i did see a small speed up in compilation, but it was taking so long to run that I left it for now, would like to revisit sometime.